app/views/categories/_category.html.erb

```diff
@@ -0,0 +1,11 @@
+<div id="<%= dom_id category %>">
+  <div class="card m-2">
+    <div class="card-body">
+      <div class="d-flex">
+        <div class="px-4">
+          <%= link_to category.name, category %>
+        </div>
+      </div>
+    </div>
+  </div>
+</div>
```

Comment:
- [ ] Kindly I would really suggest you to follow the original tempalte design as this is one of the requirements.